packages/docusaurus-theme-classic/src/theme/DocItem/index.js

```diff
@@ -13,6 +13,7 @@ import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import DocPaginator from '@theme/DocPaginator';
 import useTOCHighlight from '@theme/hooks/useTOCHighlight';
+import EditPage from '@theme/EditPage';
```

Comment:
Just copy and paste this component, don't make a new component for this edit button. Anyway it should probably be called `EditButton` if we wanted to make it a component.